tox.ini

```diff
@@ -12,10 +12,11 @@ exclude =
     build,
```

Comment:
Please migrate this to the new standard we have set, even if in a seperate PR :)

Reply:
Yes will do.